src/mono/mono/component/debugger-agent.c

```diff
@@ -529,7 +529,7 @@ is_debugger_thread (void)
 	if (!internal)
 		return FALSE;
 
-	return internal->debugger_thread;
+	return (gboolean)internal->debugger_thread;
```

Comment:
Maybe more readable and portable to do:

return internal->debugger_thread ? TRUE : FALSE